src/foam/comics/v2/DAOCreateView.js

```diff
@@ -49,6 +58,13 @@ foam.CLASS({
       class: 'FObjectProperty',
       name: 'data'
     },
+    {
+      class: 'FObjectProperty',
+      name: 'workingData',
+      expression: function(data) {
+        return data.clone(this.__subContext__)
```

Comment:
For create, I don't think you need to have a workingData. Just have `data` and give it an expression based on the config$of.

Reply:
Done.